pkgs/tools/misc/parallel/default.nix

```diff
@@ -12,10 +14,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ perl procps ];
+  propagatedbuildInputs = extraPerlPackages;
```

Comment:
Spelled wrong and I don't think it is needs to be propagated since parallel is not a library but an application.